### ISSUE TITLE
Not possible to mock multiple `save()` calls with handleCreate

### ISSUE THREAD

**Issue by soeffing** [danielspaniel/ember-data-factory-guy#88]
I have code in which I have a loop and inside this loop I call `save()`.

Example:

```
[1,2,3].forEach(function(ribbon) {
  this.store.createRecord('award', { ribbon: ribbon }).save();
});
```

It is possible to mock one request. For instance, I can mock the first request of the loop by doing this inside my tests:

```
TestHelper.handleCreate('award'); 
// OR
TestHelper.handleCreate('award').match({ribbon: 1});
```

But the subsequent requests are not being mocked. I would expect that ``` TestHelper.handleCreate('award');  ``` mocks all the three request inside the loop but it does not. Is there a way to stub multiple request with one ```.handleCreate()``` statement?

**Comment by danielspaniel** [comment]
Hmmm .. @soeffing .. that is a good question .. you're right .. that should just work. I will investigate that and see if I can fix it.

**Comment by soeffing** [comment]
@danielspaniel thx a lot! Let me know if and how I can help you...your addon is great and we really want to use it!

**Comment by danielspaniel** [comment]
no problem @soeffing .. thanks for the offer to help out :)
 I think I can get this done today or tomorrow ..

**Comment by danielspaniel** [comment]
@soeffing .. this is now fixed in version 1.0.10 of factory guy, and thank you for bringing this bug to my attention

**Comment by soeffing** [comment]
Thank you so much!
